docs/develop/parent-to-dialog.md

```diff
@@ -1,100 +1,49 @@
 ---
-title: Passing data and messages to a dialog box from its host page
-description: 'Learn how to pass data to a dialog from the host page by using the messageChild and DialogParentMessageReceived APIs.'
-ms.date: 07/07/2020
+title: Alternative ways of passing messages to a dialog box from its host page
```

Comment:
Should there be accompanying TOC changes?

Reply:
Nope. The old title in the TOC still works.